quickassist/lookaside/access_layer/src/sample_code/performance/compression/cpa_sample_code_dc_dp.c

```diff
@@ -207,7 +207,7 @@ void dcDpCallbackFunction(CpaDcDpOpData *pOpData)
     if (CPA_DC_WDOG_TIMER_ERR == (Cpa8S)pResults->status)
     {
         PRINT_ERR("Slice hang is detected\n");
-        /* fw does not respond for any furthur requests in case of slice hang
+        /* fw does not respond for any further requests in case of slice hang
```

Comment:
Should be  "...to any further requests..."

Reply:
fixed and repushed